### PR TITLE
[ME-168 & ME-181] Lesson quiz fixes

### PR DESCRIPTION
**Pull request by BartoszGamza**

### CODE REVIEW

File: app/src/constants/radioGroupThemes.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+interface Theme {
+  textClass: string;
+  containerClass: string;
+  circleClass: string;
+}
+
+interface Themes {
+  [key: string]: Theme;
+}
```
Comment (mecies) [inline app/src/constants/radioGroupThemes.ts:9]: Wdyt about changing `interfaces` to `Type` aliases? 🤔  
Like so: 
```typescript
type Theme = {
  textClass: string;
  containerClass: string;
  circleClass: string;
}

type Themes = {
  [key: string]: Theme;
}
```
I think if we ever need to extend the `Theme` we can upgrade from `Type` and make it an as `interface`. 
Does that sound good to you? 😄

File: app/src/views/subjects/[slug]/lesson/_id.tsx [hunk 2]
```diff
@@ -73,9 +73,13 @@ function LessonPage() {
     return results.find((result) => result.question_uuid === questionUuid);
   };
 
+  const isCorrect = (questionUuid: string) => (questionChoice: QuestionChoice): boolean => {
+    const questionResult = results.find((result) => result.question_uuid === questionUuid);
+    return !!questionResult?.correct_answers.includes(questionChoice.answer);
```
Comment (mecies) [inline app/src/views/subjects/[slug]/lesson/_id.tsx:78]: BANG BANG! I shot you down 💀  (all good here btw 😄 )

File: app/src/views/subjects/[slug]/lesson/_id.tsx [hunk 3]
```diff
@@ -174,7 +182,7 @@ function LessonPage() {
             <div key={test.uuid}>{renderQuestions(test.questions)}</div>
           ))}
           {error && <div className="mb-6 text-red-300">{error}</div>}
-          {tests?.results?.length && (
+          {tests?.results?.length ? (
```
Comment (mecies) [inline app/src/views/subjects/[slug]/lesson/_id.tsx:185]: I don't think that this change is needed, in this case, they work the same, and imo `&&` is prettier (does not leave a redundant `null` at the end), wdyt? 🎨  😄

Comment (BartoszGamza) [reply app/src/views/subjects/[slug]/lesson/_id.tsx:185]: It was generating that 0 (from length) we want to get rid of 😉

File: app/src/constants/radioGroupThemes.ts (hunk 1 above)
Comment (BartoszGamza) [reply app/src/constants/radioGroupThemes.ts:9]: Yeah, I totally had interface in mind 😅

File: app/src/views/subjects/[slug]/lesson/_id.tsx (hunk 3 above)
Comment (mecies) [reply app/src/views/subjects/[slug]/lesson/_id.tsx:185]: Yes, you are right as discussed Slack 👍 
Some information about the test not being present would be perfect 👌

**Comment by BartoszGamza** [comment]
@kulczewicz yes, they need to be adapted, working on it already 😉

**Pull request merged**